sdk/src/main/scala/io/horizen/account/fork/Version1_3_0Fork.scala

```diff
@@ -0,0 +1,13 @@
+package io.horizen.account.fork
+
+import io.horizen.fork.{ForkManager, OptionalSidechainFork}
+
+case class Version1_3_0Fork(active: Boolean = false) extends OptionalSidechainFork
+
+object Version1_3_0Fork {
```

Comment:
please add a comment to explain what this fork enables
(see Version1_2_0Fork for reference)

Reply:
Done